process_jartic_archives_fast.py

```
@@ -0,0 +1,217 @@
+                month = int(parts[3])
+                archive_date = datetime(year, month, 1, tzinfo=timezone.utc)
+                
+                if (archive_date.year == args.start.year and archive_date.month >= args.start.month) or \
+                   (archive_date.year == args.end.year and archive_date.month <= args.end.month) or \
+                   (args.start.year < archive_date.year < args.end.year):
+                    filtered_archives.append(archive)
+            except:
+                continue
+    
+    if not filtered_archives:
+        print(f"❌ No archives found for date range")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)